src/fragments/lib/auth/js/getting-started.mdx

```diff
@@ -31,6 +42,95 @@ Now, the authentication service has been deployed and you can start using it. To
 ```bash
 amplify console
 ```
+</Block>
+<Block name="Existing resources">
+
+If you want to re-use an existing authentication resource from AWS (e.g. Amazon Cognito UserPool or Identity Pool), update `Amplify.configure()` method with the following information.
```

Comment:
@tannerabread, that's a great callout.  Would the best solution be to move the [#Configure you application](https://docs.amplify.aws/lib/auth/getting-started/q/platform/js/#configure-your-application) section to be included WITHIN the switcher for the "Amplify CLI" block?

Reply:
That was my thought, just watch that if you do it this way that section also included the actual install of the amplify package so that would need to be included either before the block switcher (seems better to me) or _also_ in the existing resources section.

Also noticed that I was originally checking this under React Native selector and the JS page has the pre-built UI components which also include the install of the amplify package and `Amplify.configure(awsExports)` so maybe all the configuring could be handled in that top section and the installs elsewhere. Additionally with these UI sections, they import the resources as `awsExports` while the #Configure your application section called it `awsconfig`. Additionally additionally those 4 block switches are inconsistent with each other, not sure if that's something you'd want to handle in this PR

Reply:
@tannerabread, that's a valid point and something I've seen as well.  Since we're doing `export default` from the `aws-exports.js` it won't matter as much, but agreed that our naming convention should be consistent within our docs to avoid confusion.  

Maybe that aspect can be addressed with a separate PR?  I can include it within this one but for now, I'll make minimal updates per the advice above on the BlockSwitcher including the "configure your application" steps.  As far as whether or not it should be above the BlockSwitcher, @abdallahshaban557 is there any preference here? 

I'd agree that having the `npm install aws-amplify` command could be above, but we'd need to just add some different verbiage around the `Amplify.configure()` part for CLI block.

Reply:
Updated to include the "Existing Amazon resources..." verbiage of Push Notifications PR and also lifted the Config/Dependencies that were within the Blockswitcher to be above the switcher now.  There's a minimal sentence with link to the "Install Amplify Libraries" mirror the Push Notification PR as well. 

@abdallahshaban557, I put it ABOVE the Blockswitcher so that a customer isn't running through this section and then doesn't see the "Note that before..." until the end.  By placing it above the switcher, it also eliminates the text being duplicated in both blocks of the switcher.